File: fabric/src/main/java/com/kenza/thaumcraft/ThaumcraftFabric.java
package com.kenza.thaumcraft;

import com.kenza.discholder.RefKt;
import net.fabricmc.api.ModInitializer;


public class ThaumcraftFabric implements ModInitializer {
    @Override
    public void onInitialize() {

        RefKt.commonPlatformHelper = new CommonPlatformHelperFabric();
//        ModRegistries.INSTANCE.onInit();
//        ModRegistries.INSTANCE.onSetupEvent(new Object());
//        FabricSpecificImpl.INSTANCE.initData();

    }
}

File: fabric/src/main/java/com/kenza/thaumcraft/ThaumcraftFabric.kt
package com.kenza.thaumcraft

import com.turtlearmymc.doublejump.Doublejump
import io.github.cottonmc.cotton.gui.impl.LibGuiCommon
import net.fabricmc.api.ModInitializer

class ThaumcraftFabric : ModInitializer {
    override fun onInitialize() {
        commonPlatformHelper = CommonPlatformHelperFabric()

        ThaumcraftCommon.onInitialize()

        //        ModRegistries.INSTANCE.onInit();
//        ModRegistries.INSTANCE.onSetupEvent(new Object());
//        FabricSpecificImpl.INSTANCE.initData();
    }
}